.github/workflows/nodejs.yml

```
@@ -4,61 +4,60 @@
 name: Node.js CI
 
 on:
-  - push
-  - pull_request
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

```suggestion
    branches: "*"
```

So we get CI when someone open a PR against a feature branch.